lib/src/cli.rs

```diff
@@ -512,6 +515,27 @@ async fn usroverlay() -> Result<()> {
         .into());
 }
 
+/// Implementation of `bootc build commit`
+/// async fn lint() -> Result<()> {
+fn lint() -> Result<()> {
+    // TODO: Check if 1 or more folder under the /lib/modules/ directory
+    println!("Checking if we are in the container");
+    if ostree_ext::container_utils::is_ostree_container()? {
+        let modules_path = cap_std::fs::Dir::open_ambient_dir("lib/modules", cap_std::ambient_authority())?;
```

Comment:
This is a super minor nit, but canonically `lib` :arrow_right: `usr/lib` now, so slightly cleaner to use that.

Reply:
Ah ok, thank you!